laundry/models.py

```diff
@@ -0,0 +1,25 @@
+import uuid
+
+from django.db import models
+
+
+class LaundrySnapshot(models.Model):
+    hall_id = models.IntegerField()
+    date = models.DateTimeField(auto_now_add=True)
+    available_washers = models.IntegerField()
+    available_dryers = models.IntegerField()
+    total_washers = models.IntegerField()
+    total_dryers = models.IntegerField()
+
+    def __str__(self):
+        return f"Hall No. {self.hall_id} | {self.date.date()}"
+
+
+class LaundryRoom(models.Model):
```

Comment:
Could you add a comment here describing why we have a uuid field? Just so we remember what it's for at a glance